stdlib/3/os/__init__.pyi

```diff
@@ -319,7 +319,7 @@ def mkdir(path: _PathType, mode: int = ...) -> None: ...
 def makedirs(path: _PathType, mode: int = ...,
              exist_ok: bool = ...) -> None: ...
 def pathconf(path: _PathType, name: Union[str, int]) -> int: ...  # Unix only
-def readlink(path: AnyStr) -> AnyStr: ...
+def readlink(path: _PathType) -> AnyStr: ...
```

Comment:
This is a little subtle, but `_PathType` is actually generic over `AnyStr`. And you'll need that for the return type. You can try to fix this by adding `_PathType[AnyStr]`. Hopefully that fixes the issue (which is actually caused by the inferred return type of `readlink()` being broken by to this change).